CHANGELOG.md

```diff
@@ -2,7 +2,10 @@
 
 ## X.X.X (YYYY-MM-DD)
 
-
+-   Parser improvements. The following expressions are now correctly parsed:
+    -   attributes at the end of `Condition` instances;
+    -   attributes of `Rule` instances;
+    -   order one conditions of `whatever` expressions.
```

Comment:
Whaaaat? xD

Reply:
Is `whatever` a YML keyword?

Reply:
Yes :) 
![image](https://user-images.githubusercontent.com/15327919/135635367-ce0add04-24cb-4880-a90c-826c25b70e48.png)